ingenialink/ethernet/dictionary.py

```diff
@@ -0,0 +1,53 @@
+from ingenialink.dictionary import Dictionary
+from ingenialink.ethernet.register import EthernetRegister
+
+import ingenialogger
+
+logger = ingenialogger.get_logger(__name__)
+
+
+class EthernetDictionary(Dictionary):
+    """Contains all registers and information of a Ethernet dictionary.
+
+    Args:
+        dictionary_path (str): Path to the Ingenia dictionary.
+
+    """
+
+    class AttrRegEthDict(Dictionary.AttrRegDict):
+        ADDR = 'address'
+
+    def __init__(self, dictionary_path):
+        super().__init__(dictionary_path)
+
+    def _read_xdf_register(self, register):
+        """Reads a register from the dictionary and creates a Register instance.
+
+        Args:
+            register (Element): Register instance from the dictionary.
```

Comment:
Add in the docstring the return documentation. But first, check if it can be defined only in the parent function, and child function inheritances the docstring, In this case, remove the child docstring to avoid writing it twice.